.scripts/util.sh

```diff
@@ -59,25 +63,61 @@ is_generated() {
     fi
 }
 
+select_browser_caller() {
+    if [ -z "$1" ]; then
+        BROWSER="chromium"
+        echo "The default browser '$BROWSER' was selected as no option was selected manually."
+    elif [ "$1" = "chromium" ]; then
+        BROWSER="chromium"
+        SYS_OPTS="-DBROWSER_INCLUDE_DIR=$PACKAGE_DIR/chromiumembeddedframework.runtime.$CHROMIUM_VERSION/build/native/include -DBROWSER_LIB_DIR=$PACKAGE_DIR/chromiumembeddedframework.runtime.$CHROMIUM_VERSION/build/native/x64 -DBROWSER=$BROWSER"
```

Comment:
We can format these for readability as:
```bash
    SYS_OPTS="\
-DBROWSER_INCLUDE_DIR=$PACKAGE_DIR/chromiumembeddedframework.runtime.$CHROMIUM_VERSION/build/native/include \
-DBROWSER_LIB_DIR=$PACKAGE_DIR/chromiumembeddedframework.runtime.$CHROMIUM_VERSION/build/native/x64 \
-DBROWSER=$BROWSER"
```
In `generate` we use `$SYS_OPTS` unquoted for expansion and suppress the warning with `# shellcheck disable=SC2086`.